FogCreek/Glitch-Community
# Cassey-community: a11y small fixes

## Links
* cassey-community.glitch.me
* FogCreek/Glitch-Community-Work#139
* FogCreek/Glitch-Community-Work#134
* FogCreek/Glitch-Community-Work#162
* FogCreek/Glitch-Community-Work#140
* FogCreek/Glitch-Community-Work#142

## Changes:
* Make sure the More Ideas section is consistently outside of `<main>` and in an `<aside>`
* Add more descriptive alt text to the verified badge that shows next to a team/in search results
* Add a visually hidden header on the secrets page since the text is in the background image
* Make the dev toggles buttons on the secrets page into toggle buttons for assistive tech users

## How To Test:
* Go to the dev toggles page and toggle some stuff to make sure I didn't break anything there; make sure it looks good too

## Feedback I'm looking for:
Any unexpected effects of these changes?
